diff --git a/pl_bolts/models/self_supervised/moco/moco2_module.py b/pl_bolts/models/self_supervised/moco/moco2_module.py
index f963925..6d54053 100644
--- a/pl_bolts/models/self_supervised/moco/moco2_module.py
+++ b/pl_bolts/models/self_supervised/moco/moco2_module.py
@@ -326,7 +326,6 @@ class MocoV2(pl.LightningModule):
         parser.add_argument('--base_encoder', type=str, default='resnet18')
         parser.add_argument('--emb_dim', type=int, default=128)
         parser.add_argument('--num_workers', type=int, default=psutil.cpu_count())
-        parser.add_argument('--gpus', type=int, default=-1)
         parser.add_argument('--num_negatives', type=int, default=65536)
         parser.add_argument('--encoder_momentum', type=float, default=0.999)
         parser.add_argument('--softmax_temperature', type=float, default=0.07)
@@ -336,10 +335,8 @@ class MocoV2(pl.LightningModule):
         parser.add_argument('--data_dir', type=str, default='./')
         parser.add_argument('--dataset', type=str, default='cifar10', help='cifar10, stl10, imagenet2012')
         parser.add_argument('--batch_size', type=int, default=256)
-        parser.add_argument('--max_epochs', type=int, default=1000)
         parser.add_argument('--use_mlp', action='store_true')
         parser.add_argument('--meta_dir', default='.', type=str, help='path to meta.bin for imagenet')
-        parser.add_argument('--logger', default=WandbLogger(project="MocoV2"))
 
         return parser
 
@@ -393,8 +390,8 @@ def cli_main():
         datamodule = None
 
     model = MocoV2(**args.__dict__)
-
-    trainer = pl.Trainer.from_argparse_args(args)
+    wandb_logger = WandbLogger(name='Baseline', project='MocoV2')
+    trainer = pl.Trainer.from_argparse_args(args, logger = wandb_logger)
     trainer.fit(model, datamodule=datamodule)
     wandb.finish()
 
